Tasks/AzureRmWebAppDeployment/CompressionUtility.ps1

```diff
@@ -0,0 +1,48 @@
+﻿function UnzipWebDeployPkg
```

Comment:
@ericsciple We are planning to support other compression formats in future e.g. war , tar.gz . That's why we are taking dependency on 7zip.